### DISCUSSION TITLE
Mining on Old iPhones....

### DISCUSSION THREAD

**Discussion by itsmeherejim** [catchthatrabbit/coreminer#40] [Hardware]
Fellow Rabbits,
I see that Ubuntu can now be loaded on an iPhone or iPad. Is it possible to turn old phones into miners? On this holiday weekend I'm planning to give-it-a-go on my old phones. Any suggestions? Or should I just go skiing? (Yes there should be new snow). Cheers, Jim

**Comment by rastislavcore** [comment]
Not tested officialy yet. Please, feel free to test it and we will be happy to receive any info to improve the app.

**Comment by emanuelnuta** [comment]
Hi Rasty,
could you help me in a problem ???
i have a DELL Server R620 ( 1x Intel DECA Core Xeon  E5-2670 v2 2.50GHz (3.30GHz Turbo, 25MB Cache)) and my mining KH/s is maximum 5.75.
this server has a lot of processors !! could you explain what happend or is a setting in Coreminer ??
thank a lot !!!

**Reply by rastislavcore** [reply]
It all depends on your configuration. Can you share your full spec including OS?

**Comment by emanuelnuta** [comment]
please find attached. it is Dell T620 server with WIN11 OS.

*  EMANUEL NUTA  * 

    On Monday, July 18, 2022 at 11:31:21 PM GMT+3, Rastislav ***@***.***> wrote:  
 
 


It all depends on your configuration. Can you share your full spec including OS?

**Reply by rastislavcore** [reply]
Only Linux with Glibc 2.33 and more are supported now.
See more here https://github.com/catchthatrabbit/coreminer#limitations

**Comment by Schoenmakerr** [comment]
Hi, is it possible on an old iPhone?

**Comment by itsmeherejim** [comment]
Rasti: Will do. I wonder how may battered old  iPhones are sitting drawers gathering dust? (I have three.) This is a long shot, but possibly a good learning-exercise for me.  Certainly not a core priority for Core!  But I will report. 
Go green! Cheers. 

Jim
--
James N. Hansen
***@***.***

65380 Olson Rd
Deer Island, OR 97054
ph. 503.307.3164
